clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -368,7 +368,7 @@ def err_drv_fsycl_with_pch : Error<
 def err_drv_fsycl_unsupported_with_opt
   : Error<"'%0' is not supported with '-fsycl'">;
 def warn_drv_fsycl_host_compiler_options
-  : Warning<"'%0' should be used only in conjunction with '-fsycl-host-compiler'">, InGroup<InvalidCommandLineArgument>;
+  : Warning<"'%0' should be used only in conjunction with '%1'">, InGroup<UnusedCommandLineArgument>;
```

Comment:
Please update the value of the diagnostic to match (no longer directly associated with fsycl_host_compiler)